plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -105,30 +106,32 @@ def _onWriteStarted(self, output_device):
             material_used = [str(math.pi * material_radius * material_radius * material_length) for material_length in print_information.materialLengths]
             material_used = ",".join(material_used)
 
+            containers = { "": global_container_stack.serialize() }
+            for container in global_container_stack.getContainers():
+                container_id = container.getId()
+                container_serialized = ""
+                try:
+                    container_serialized = container.serialize()
+                except NotImplementedError:
+                    Logger.log("w", "Container %s could not be serialized!", container_id)
+                    continue
+                if container_serialized:
+                    containers[container_id] = container_serialized
+                else:
+                    Logger.log("i", "No data found in %s to be serialized!", container_id)
+
             # Bundle the collected data
             submitted_data = {
                 "processor": platform.processor(),
                 "machine": platform.machine(),
                 "platform": platform.platform(),
-                "settings": global_container_stack.serialize(), # global_container with references on used containers
+                "settings": json.dumps(containers), # global_container with references on used containers
```

Comment:
This is done with https://github.com/Ultimaker/Cura/pull/1015/commits/10d2d7902ea35a135afe249791305a8c447b6d1d